### PR TITLE
Update PowerDNS.com.xml

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: src/chrome/content/rules/PowerDNS.com.xml [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
+		- xs.powerdns.com
 
+	Secure connection failed:
+		- pdnsdev.powerdns.com
 -->
-<ruleset name="PowerDNS.com" default_off='failed ruleset test'>
+<ruleset name="PowerDNS.com">
 
 	<target host="powerdns.com" />
+	<target host="www.powerdns.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/PowerDNS.com.xml]: blog.powerdns.com

File: src/chrome/content/rules/PowerDNS.com.xml [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+	<target host="builder.powerdns.com" />
 	<target host="doc.powerdns.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/PowerDNS.com.xml]: docs.powerdns.com

File: src/chrome/content/rules/PowerDNS.com.xml [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
 	<target host="downloads.powerdns.com" />
-	<target host="rt.powerdns.com" />
-	<target host="www.powerdns.com" />
-	<target host="xs.powerdns.com" />
+	<target host="mailman.powerdns.com" />
+	<target host="metronome1.powerdns.com" />
+	<target host="repo.powerdns.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/PowerDNS.com.xml]: rtfm.powerdns.com

**Comment by J0WI** [comment]
Thanks, I've added the remaining targets.

**Pull request merged**